File: Java/src/CCC_J2/rsaNumbers.java
package CCC_J2;
import java.util.Scanner;
public class rsaNumbers {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int lowerBound = scan.nextInt();
        int upperBound = scan.nextInt();
        int divisors = 0;
        int total = 0;
        for (int i = lowerBound; i <= upperBound; i++){
            for (int k = 1; k <= i; k++){
                if (i % k == 0){
                    divisors ++;
                }
            }
            if(divisors == 4){
                total += 1;
            }
            divisors = 0;
        }
        System.out.println("The number of RSA numbers between " + String.valueOf(lowerBound) + " and " + String.valueOf(upperBound) + " is " + String.valueOf(total));
    }
}

File: CCC/J2/rsaNumbers.py
from_num = int(input())
to_num = int(input())
counter = 0
store = 0
RSA_Numbers = 0

for i in range(from_num, to_num + 1):
    counter = 0
    for l in range(1, i + 1):
        store = i % l
        if store == 0:
            counter += 1
    if counter == 4:
        RSA_Numbers += 1

print("The number of RSA numbers between", from_num, "and", to_num, "is", RSA_Numbers)